packages/docs-site/src/content/docs/core-concepts/what-is-taiko-alethia.md

```diff
@@ -3,59 +3,98 @@ title: What is Taiko Alethia?
+| --------------------------- | ---------------------------------------------------------------------------------------- |
+| **Taiko Community**         | Open social groups and discussions (Discord, Twitter, forums).                           |
+| **Taiko Labs**              | Research and development entity supporting the Taiko Alethia protocol.                   |
+| **Taiko Treasury**          | Collects fees from L2 congestion pricing and distributes funds for development.          |
+| **Taiko DAO (in progress)** | Governing body managing smart contract upgrades, network parameters, and protocol funds. |
+| **Taiko Foundation**        | Funds technical development, partnerships, and ecosystem growth.                         |
+| **Taiko Security Council**  | Handles critical protocol security, Guardian Provers, and emergency network decisions.   |
+
+---
+
+## Infrastructure Operated by Taiko Labs
+
+Taiko Labs operates non-critical and critical infrastructure, but anyone can run these components due to Taiko’s open-source and permissionless nature.
+
+### Non-Critical Infrastructure
 
-:::note
-Anyone can run these components, not just Taiko Labs. Yes you can sequence blocks on Taiko, host your own bridge using our [signal service](/taiko-alethia-protocol/bridging#the-signal-service), etc.
-:::
+These services are open-source, meaning anyone can replicate or improve them.
 
 #### Frontends
 
-- [Bridge UI](https://bridge.taiko.xyz)
-- [Network status page](https://status.taiko.xyz)
-- [Main homepage](https://taiko.xyz) + [this documentation site](https://docs.taiko.xyz)
-- [Geth fork diff page](https://geth.taiko.xyz)
+- [Bridge UI](https://bridge.taiko.xyz) → Interface for asset transfers between L1 & L2.
+- [Network status](https://status.taiko.xyz) → Live updates on Taiko Alethia.
+- [Homepage & Documentation](https://taiko.xyz) → Official website and developer resources.
+- [Geth fork diff](https://geth.taiko.xyz) → Fork comparison for Ethereum-equivalence.
 
 #### Backends
 
-- [Event indexer](/api-reference/event-indexer)
-- [Bridge relayer](/api-reference/bridge-relayer)
-- [Taiko Alethia P2P bootstrapping nodes](/network-reference/alethia-addresses#taiko-labs-bootnode-addresses)
-- [Taiko Alethia Taiko Labs' proposers and provers](/network-reference/alethia-addresses)
-- [Taiko Hekla P2P bootstrapping nodes](/network-reference/hekla-addresses#taiko-labs-bootnode-addresses)
-- [Taiko Hekla Taiko Labs' proposer and provers](/network-reference/hekla-addresses)
+- [Event Indexer](/api-reference/event-indexer) → Tracks rollup transactions & events.
+- [Bridge Relayer](/api-reference/bridge-relayer) → Facilitates trust-minimized bridging.
+- **Taiko Alethia & Hekla P2P Bootstrapping Nodes** → Helps decentralized peers sync with the network.
+- **Taiko Alethia & Hekla Proposers & Provers** → Supports decentralized block production.
```

Comment:
this should link to our proposer and prover addresses no? I think we can have it like we did before @swarna1101